### ISSUE TITLE
Version 0.5 fails to parse tasks file (and refuses to launch) when a saved search's name and query matches exactly one of the tags

### ISSUE THREAD

**Issue by iaeiou** [getting-things-gnome/gtg#629]
Hello there,

I was using GTG 0.4 with Flatpak and everything was going smoothly.

Today the new version (0.5) is running after an automatic update. Unfortunately, only the test tasks are showing now:

![gtg2](https://user-images.githubusercontent.com/69427615/114487761-71590000-9c10-11eb-963e-1e90975a355c.png)

I think there's no data loss though, here is the treeview of  `~/.var/app/org.gnome.GTG`

```
.
├── cache
│   ├── fontconfig
│   │   └── 8e575bbd690cbdd5c13d70f46836611e-le64.cache-7
│   ├── mesa_shader_cache
│   │   ├── 7b
│   │   │   └── 129dd520072e893669cbd79084012ead13f847
│   │   ├── 8c
│   │   │   └── 96c84181837fef71d69bec0d80f4461654e67d
│   │   ├── 93
│   │   │   └── b39334786cf727f6b3c2fccda51462ab461af6
│   │   ├── 9d
│   │   │   └── 24b54391ecbab9c3cdda96131158b038beae83
│   │   ├── a9
│   │   │   └── aa11003a3cf6237ae869d3a085ddb9664cab72
│   │   ├── ab
│   │   │   └── f7769e64fc012c17c312fce2667f6bc479cf6b
│   │   ├── dc
│   │   │   └── a844c1628e39756e9c218b18b4538519f7a7dd
│   │   ├── de
│   │   │   └── 43b818d9980a0a4bdcc07e913b3bc467359b0c
│   │   └── index
│   ├── tmp
│   └── yelp
│       └── WebKitCache
│           └── Version 16
│               ├── Blobs
│               └── salt
├── config
│   ├── glib-2.0
│   │   └── settings
│   │       └── keyfile
│   ├── gtg
│   │   ├── backends.conf
│   │   ├── gtg.conf
│   │   ├── plugins
│   │   │   └── Urgency Color
│   │   │       └── preferences
│   │   └── tasks.conf
│   └── yelp
└── data
    ├── gtg
    │   ├── backup
    │   │   ├── gtg_data.xml.2021-04-13.bak
    │   │   ├── gtg_data.xml.bak.0
    │   │   ├── gtg_data.xml.bak.1
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-11.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-12.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-19.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-20.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-21.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-22.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-23.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-25.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-26.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-27.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-28.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-30.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-01-31.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-01.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-02.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-03.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-05.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-06.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-07.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-10.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-11.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-12.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-13.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-16.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-19.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-23.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-24.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-02-27.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-01.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-06.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-07.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-12.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-13.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-14.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-18.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-24.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-25.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-28.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-03-29.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-04-05.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2021-04-06.bak
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.0
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.1
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.2
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.3
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.4
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.5
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.6
    │   │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.7
    │   │   ├── projects.xml.2021-01-11.bak
    │   │   ├── projects.xml.2021-01-12.bak
    │   │   ├── projects.xml.2021-01-19.bak
    │   │   ├── projects.xml.2021-01-20.bak
    │   │   ├── projects.xml.2021-01-21.bak
    │   │   ├── projects.xml.2021-01-22.bak
    │   │   ├── projects.xml.2021-01-23.bak
    │   │   ├── projects.xml.2021-01-25.bak
    │   │   ├── projects.xml.2021-01-26.bak
    │   │   ├── projects.xml.2021-01-27.bak
    │   │   ├── projects.xml.2021-01-28.bak
    │   │   ├── projects.xml.2021-01-30.bak
    │   │   ├── projects.xml.2021-01-31.bak
    │   │   ├── projects.xml.2021-02-01.bak
    │   │   ├── projects.xml.2021-02-02.bak
    │   │   ├── projects.xml.2021-02-05.bak
    │   │   ├── projects.xml.2021-02-06.bak
    │   │   ├── projects.xml.2021-02-07.bak
    │   │   ├── projects.xml.2021-02-10.bak
    │   │   ├── projects.xml.2021-02-11.bak
    │   │   ├── projects.xml.2021-02-13.bak
    │   │   ├── projects.xml.2021-02-16.bak
    │   │   ├── projects.xml.2021-02-19.bak
    │   │   ├── projects.xml.2021-02-23.bak
    │   │   ├── projects.xml.2021-02-24.bak
    │   │   ├── projects.xml.2021-02-27.bak
    │   │   ├── projects.xml.2021-03-01.bak
    │   │   ├── projects.xml.2021-03-06.bak
    │   │   ├── projects.xml.2021-03-07.bak
    │   │   ├── projects.xml.2021-03-12.bak
    │   │   ├── projects.xml.2021-03-13.bak
    │   │   ├── projects.xml.2021-03-14.bak
    │   │   ├── projects.xml.2021-03-18.bak
    │   │   ├── projects.xml.2021-03-20.bak
    │   │   ├── projects.xml.2021-03-24.bak
    │   │   ├── projects.xml.2021-03-25.bak
    │   │   ├── projects.xml.2021-03-26.bak
    │   │   ├── projects.xml.2021-03-28.bak
    │   │   ├── projects.xml.2021-03-29.bak
    │   │   ├── projects.xml.2021-03-30.bak
    │   │   ├── projects.xml.2021-04-05.bak
    │   │   ├── projects.xml.2021-04-06.bak
    │   │   ├── projects.xml.bak.0
    │   │   ├── projects.xml.bak.1
    │   │   ├── projects.xml.bak.2
    │   │   ├── projects.xml.bak.3
    │   │   ├── projects.xml.bak.4
    │   │   ├── projects.xml.bak.5
    │   │   ├── projects.xml.bak.6
    │   │   ├── projects.xml.bak.7
    │   │   ├── tags.xml.2021-01-11.bak
    │   │   ├── tags.xml.2021-01-12.bak
    │   │   ├── tags.xml.2021-01-19.bak
    │   │   ├── tags.xml.2021-01-20.bak
    │   │   ├── tags.xml.2021-01-21.bak
    │   │   ├── tags.xml.2021-01-22.bak
    │   │   ├── tags.xml.2021-01-23.bak
    │   │   ├── tags.xml.2021-01-25.bak
    │   │   ├── tags.xml.2021-01-26.bak
    │   │   ├── tags.xml.2021-01-27.bak
    │   │   ├── tags.xml.2021-01-28.bak
    │   │   ├── tags.xml.2021-01-30.bak
    │   │   ├── tags.xml.2021-01-31.bak
    │   │   ├── tags.xml.2021-02-01.bak
    │   │   ├── tags.xml.2021-02-02.bak
    │   │   ├── tags.xml.2021-02-05.bak
    │   │   ├── tags.xml.2021-02-06.bak
    │   │   ├── tags.xml.2021-02-07.bak
    │   │   ├── tags.xml.2021-02-10.bak
    │   │   ├── tags.xml.2021-02-11.bak
    │   │   ├── tags.xml.2021-02-13.bak
    │   │   ├── tags.xml.2021-02-16.bak
    │   │   ├── tags.xml.2021-02-19.bak
    │   │   ├── tags.xml.2021-02-23.bak
    │   │   ├── tags.xml.2021-02-24.bak
    │   │   ├── tags.xml.2021-02-27.bak
    │   │   ├── tags.xml.2021-03-01.bak
    │   │   ├── tags.xml.2021-03-06.bak
    │   │   ├── tags.xml.2021-03-07.bak
    │   │   ├── tags.xml.2021-03-12.bak
    │   │   ├── tags.xml.2021-03-13.bak
    │   │   ├── tags.xml.2021-03-14.bak
    │   │   ├── tags.xml.2021-03-18.bak
    │   │   ├── tags.xml.2021-03-20.bak
    │   │   ├── tags.xml.2021-03-24.bak
    │   │   ├── tags.xml.2021-03-25.bak
    │   │   ├── tags.xml.2021-03-26.bak
    │   │   ├── tags.xml.2021-03-28.bak
    │   │   ├── tags.xml.2021-03-29.bak
    │   │   ├── tags.xml.2021-03-30.bak
    │   │   ├── tags.xml.2021-04-05.bak
    │   │   ├── tags.xml.2021-04-06.bak
    │   │   ├── tags.xml.bak.0
    │   │   ├── tags.xml.bak.1
    │   │   ├── tags.xml.bak.2
    │   │   ├── tags.xml.bak.3
    │   │   ├── tags.xml.bak.4
    │   │   ├── tags.xml.bak.5
    │   │   ├── tags.xml.bak.6
    │   │   └── tags.xml.bak.7
    │   ├── gtg_data.xml
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-09.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-12.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-16.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-24.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-25.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-26.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-27.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-28.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-29.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-30.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-10-31.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-02.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-04.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-05.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-08.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-09.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-10.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-11.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-12.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-16.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-17.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-18.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-20.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-21.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-22.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-25.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-26.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-27.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-29.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-11-30.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-01.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-03.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-04.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-08.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-10.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-11.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-12.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-17.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-18.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-19.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-22.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2015-12-23.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2016-01-13.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2016-01-14.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2016-01-15.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2016-01-16.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.2016-01-20.bak
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.0
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.1
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.2
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.3
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.4
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.5
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.6
    │   ├── gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml.bak.7
    │   ├── gtg_tasks-backup du fichier d'origine.xml
    │   ├── projects.xml
    │   ├── projects.xml.2015-10-09.bak
    │   ├── projects.xml.2015-10-12.bak
    │   ├── projects.xml.2015-10-16.bak
    │   ├── projects.xml.2015-10-24.bak
    │   ├── projects.xml.2015-10-25.bak
    │   ├── projects.xml.2015-10-26.bak
    │   ├── projects.xml.2015-10-27.bak
    │   ├── projects.xml.2015-10-28.bak
    │   ├── projects.xml.2015-10-29.bak
    │   ├── projects.xml.2015-10-30.bak
    │   ├── projects.xml.2015-10-31.bak
    │   ├── projects.xml.2015-11-02.bak
    │   ├── projects.xml.2015-11-04.bak
    │   ├── projects.xml.2015-11-05.bak
    │   ├── projects.xml.2015-11-08.bak
    │   ├── projects.xml.2015-11-09.bak
    │   ├── projects.xml.2015-11-10.bak
    │   ├── projects.xml.2015-11-11.bak
    │   ├── projects.xml.2015-11-12.bak
    │   ├── projects.xml.2015-11-16.bak
    │   ├── projects.xml.2015-11-17.bak
    │   ├── projects.xml.2015-11-18.bak
    │   ├── projects.xml.2015-11-20.bak
    │   ├── projects.xml.2015-11-21.bak
    │   ├── projects.xml.2015-11-22.bak
    │   ├── projects.xml.2015-11-25.bak
    │   ├── projects.xml.2015-11-26.bak
    │   ├── projects.xml.2015-11-27.bak
    │   ├── projects.xml.2015-11-29.bak
    │   ├── projects.xml.2015-11-30.bak
    │   ├── projects.xml.2015-12-01.bak
    │   ├── projects.xml.2015-12-03.bak
    │   ├── projects.xml.2015-12-04.bak
    │   ├── projects.xml.2015-12-08.bak
    │   ├── projects.xml.2015-12-10.bak
    │   ├── projects.xml.2015-12-11.bak
    │   ├── projects.xml.2015-12-12.bak
    │   ├── projects.xml.2015-12-17.bak
    │   ├── projects.xml.2015-12-18.bak
    │   ├── projects.xml.2015-12-19.bak
    │   ├── projects.xml.2015-12-22.bak
    │   ├── projects.xml.2015-12-23.bak
    │   ├── projects.xml.2016-01-13.bak
    │   ├── projects.xml.2016-01-14.bak
    │   ├── projects.xml.2016-01-15.bak
    │   ├── projects.xml.2016-01-16.bak
    │   ├── projects.xml.2016-01-20.bak
    │   ├── projects.xml.bak.0
    │   ├── projects.xml.bak.1
    │   ├── projects.xml.bak.2
    │   ├── projects.xml.bak.3
    │   ├── projects.xml.bak.4
    │   ├── projects.xml.bak.5
    │   ├── projects.xml.bak.6
    │   ├── projects.xml.bak.7
    │   └── tags.xml
    └── webkitgtk
        ├── deviceidhashsalts
        │   └── 1
        └── localstorage

30 directories, 283 files
```

Any hint, or additional way to report relevant infos?

**Comment by diegogangl** [comment]
Did you go from 0.3 to 0.4 and then 0.5? It's a versioning thing, since we have a new file format. I didn't expect many people to still have the 0.3 style filenames :)

Try this:

- Delete `gtg_data.xml`
- Rename `gtg_tasks-b6862461-76ba-492c-9adb-43ccd4b08af6.xml` to `gtg_tasks.xml`

Try running gtg again

**Comment by iaeiou** [comment]
Hi @diegogangl, thank you for your answer,

I did what you mentioned, and started `gtg` again:

```
$ flatpak run org.gnome.GTG 
GNOME keyring not found, passwords will be not stored after restarting GTG
```

`gnome-keyring` is already installed (but is it accessible to Flatpak packages?):

```
$ sudo apt install gnome-keyring
gnome-keyring is already the newest version (3.28.2-5).
```

I'm totally new to Flatpak, I tried the following commands:

```
$ flatpak run --command=sh org.gnome.GTG 
sh: parse_git_branch: command not found
```

```
flatpak run --devel --command=sh org.gnome.GTG 
error: runtime/org.gnome.Sdk/x86_64/40 not installed
```

**Comment by diegogangl** [comment]
That message is harmless since GTG isn't storing any passwords ATM. Is the window not coming up when you run the flatpak?

**Comment by iaeiou** [comment]
> Is the window not coming up when you run the flatpak?

Unfortunately not. But the message in the Terminal is a bit more verbose today:

```
$ flatpak run org.gnome.GTG
GNOME keyring not found, passwords will be not stored after restarting GTG
Gtk-Message: 17:26:59.479: Failed to load module "canberra-gtk-module"
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/GTG/gtk/application.py", line 97, in do_startup
    datastore.register_backend(backend_dic)
  File "/app/lib/python3.8/site-packages/GTG/core/datastore.py", line 459, in register_backend
    source.initialize(connect_signals=False)
  File "/app/lib/python3.8/site-packages/GTG/core/datastore.py", line 822, in initialize
    self.backend.initialize()
  File "/app/lib/python3.8/site-packages/GTG/backends/backend_localfile.py", line 135, in initialize
    self.datastore.load_search_tree(self.search_tree)
  File "/app/lib/python3.8/site-packages/GTG/core/datastore.py", line 280, in load_search_tree
    self.new_search_tag(name, query, tag_attrs, tid, False)
  File "/app/lib/python3.8/site-packages/GTG/core/datastore.py", line 144, in new_search_tag
    self._add_new_tag(name, tag, search_filter, parameters,
  File "/app/lib/python3.8/site-packages/GTG/core/datastore.py", line 108, in _add_new_tag
    raise IndexError(f'tag {name} was already in the datastore')
IndexError: tag vaness was already in the datastore
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/GTG/gtk/application.py", line 123, in do_activate
    self.init_shared()
  File "/app/lib/python3.8/site-packages/GTG/gtk/application.py", line 160, in init_shared
    self.init_browser()
  File "/app/lib/python3.8/site-packages/GTG/gtk/application.py", line 168, in init_browser
    self.browser = MainWindow(self.req, self)
  File "/app/lib/python3.8/site-packages/GTG/gtk/browser/main_window.py", line 70, in __init__
    self.config = self.req.get_config('browser')
AttributeError: 'NoneType' object has no attribute 'get_config'
Traceback (most recent call last):
  File "/app/lib/python3.8/site-packages/GTG/gtk/application.py", line 123, in do_activate
    self.init_shared()
  File "/app/lib/python3.8/site-packages/GTG/gtk/application.py", line 160, in init_shared
    self.init_browser()
  File "/app/lib/python3.8/site-packages/GTG/gtk/application.py", line 168, in init_browser
    self.browser = MainWindow(self.req, self)
  File "/app/lib/python3.8/site-packages/GTG/gtk/browser/main_window.py", line 70, in __init__
    self.config = self.req.get_config('browser')
AttributeError: 'NoneType' object has no attribute 'get_config'
```

No window appears but a process is running in the background:

```
$ ps -aux |grep gtg
root      3515  0.0  0.0   5192   972 pts/2    S+   17:26   0:00 bwrap --args 36 gtg
amnesia   3540  0.0  0.0   5456   972 pts/2    S+   17:26   0:00 bwrap --args 36 gtg
amnesia   3542  2.2  0.3 262048 57720 pts/2    Sl+  17:26   0:00 python3 /app/bin/gtg
amnesia   3901  0.0  0.0   6080   884 pts/5    S+   17:27   0:00 grep --color=auto gtg
```

**Comment by Neui** [comment]
Kill the running process. Make a backup of `~/.var/app/org.gnome.GTG/data/gtg/gtg_data.xml` (copy and just name it something else, that should be enough), then open the original file in a text editor. Near the top you should see lines starting with `<tag`, and on the same line there should be `name="<some name>"`, describing the tag name. There is likely another line with the same `name="..."`. Delete one of them.

Note that some other error could occur if you delete the wrong one, since the tasks uses the id (`id="<the id>"`) would not find it by that id. In that case, you can try restoring the file and delete the other line, or rename the tag entirely (change the `name="<some name>"` the `<some name>` to something else).

(You can ignore the `AttributeError: 'NoneType' object has no attribute 'get_config'`, since GTG doesn't handle errors during initialization well and that is just a follow-up error, the real error are the lines before it)

**Comment by iaeiou** [comment]
**Fixed!**

**I had to delete a search query that was similar to a tag name**.

**More details:**

The file `~/.var/app/org.gnome.GTG/data/gtg/gtg_data.xml` contained the following lines:

```
    <tag id="f423382c-0af3-41a9-8ad6-dbafc5314f9f" name="redact"/>
    <tag id="36946aba-b456-4962-8e67-875d981fbcc8" name="portfolio"/>
  </taglist>
  <searchlist>
    <savedSearch id="3cc31b92-a269-4f10-975c-a09645872409" name="redact" query="redact" parent="search"/>
  </searchlist>
```

The problem occurs only when the `savedSearch` is similar to a `tag name`, modifying the savedSearch or removing the line fixes the issue

What I did precisely:

- killed GTG with `flatpak kill org.gnome.GTG`
- opened `~/.var/app/org.gnome.GTG/data/gtg/gtg_data.xml`
- deleted the line containing `<savedSearch`

Thanks a lot @Neui for giving me the right direction!

I let this ticket opened, as something may need to be fixed. Feel free to close if it's a corner case

**Comment by nekohayo** [comment]
Aaah, so we may finally have found what was causing these issues? Might be when a saved search matches exactly the name of a tag (a bit similar maybe to what was found in https://github.com/getting-things-gnome/gtg/issues/616#issuecomment-817135185 regarding dots/periods in tag names). Maybe @diegogangl can figure a way to detect and correct that...

**Comment by steveblamey** [comment]
I experience similar behaviour in 0.5 when editing a tag name.

I had a tag 'Open', which I changed to 'open' (i.e. changed the first letter to lower case). After closing GTG and running again it produced the errors seen in the [comment](https://github.com/getting-things-gnome/gtg/issues/629#issuecomment-821841031) above.

Looking at `~/.var/app/org.gnome.GTG/data/gtg/gtg_data.xml`, there were now two tags with the name 'open'. So an additional tag had been created on renaming the original.

**Comment by KatonaZsolt** [comment]
> I experience similar behaviour in 0.5 when editing a tag name.
> 
> I had a tag 'Open', which I changed to 'open' (i.e. changed the first letter to lower case). After closing GTG and running again it produced the errors seen in the [comment](https://github.com/getting-things-gnome/gtg/issues/629#issuecomment-821841031) above.
> 
> Looking at `~/.var/app/org.gnome.GTG/data/gtg/gtg_data.xml`, there were now two tags with the name 'open'. So an additional tag had been created on renaming the original.

Similar problem happened to me as well, and since then GTG won't start.

**Comment by nekohayo** [comment]
Retitling (for clarity) + retagging + targetting this issue. High priority because it can prevent launch and is quite confusing when it happens, not "critical" because at least there is a workaround and the file format can be somewhat easily corrected by users. I suspect this will be one of the things Diego hopes to address as part of #553 so we'll need to keep an eye on this for whenever that lands.

**Comment by Leyeev** [comment]
I have experience the same but deleting the saved search and same label tag did not work for me. It still shows the same error when launching in flatpak. I have fixed this by simply restoring backup. In which, it is weird that the backup still contains the similar tag with upper case problem, and save search, the still launches

**Comment by dmp1ce** [comment]
This got me when I had similar named tags with capitalization differences and I changed one to match the other tag. I also added an icon to one or both of the tags.

**Comment by diegogangl** [comment]
Merged @Neui 's workaround, this is also properly fixed by the new core ( #668 )
